### ISSUE TITLE
ObjectiveGit in conflict with gitx

### ISSUE THREAD

**Issue by thisgeek** [rowanj/gitx#34]
```
$ gitx
objc[8955]: Class __ARCLite__ is implemented in both /Applications/GitX.app/Contents/Resources/../Frameworks/ObjectiveGit.framework/Versions/A/ObjectiveGit and /usr/local/bin/gitx. One of the two will be used. Which one is undefined.
```

This does _not_ appear to block functionality.

**Comment by rowanj** [comment]
This is due to both GitX and Objective-git being built with Automated Reference Counting and targeting Snow Leopard.  ARCLite is the Snow Leopard compatibility library for ARC; which is linked non-optionally into each target with ARC enabled that targets Snow Leopard.

I may be able to suppress this by building the frameworks for static linking instead of dynamic, but at least rest assured that the two implementations that the linker finds at load times are identical; so there should be no problems beyond the printing of this message.

**Comment by kriskhaira** [comment]
I'm having this problem too. GitX 0.14.23, git 1.7.7.5, Lion.

**Comment by rowanj** [comment]
This will be resolved by #40
